dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -44,15 +45,14 @@ protected String parseCommandText(String commandText) {
 
     public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
             throws SQLException {
-        String[] qualifiers = NameNormaliser.normaliseName(tableOrViewName)
-                .split("\\.");
+        String[] qualifiers = tableOrViewName.split("\\.");
         String qry = " select column_name, data_type, character_maximum_length "
                 + "	as direction from information_schema.columns where ";
 
         if (qualifiers.length == 2) {
-            qry += " lower(table_schema)=? and lower(table_name)=? ";
+            qry += " table_schema=? and table_name=? ";
```

Comment:
OK... Update actually works a bit different than e.g. Insert. It's interesting to check also Insert with both upper and lower case - maybe in the same test can be split to check both scenarios:

```
|insert|users|
|name|username|
|arthur dent|adent|

|INSERT|USERS|
|NAME|USERNAME|
|ford prefect|fpref|
|zaphod beeblebrox|zaphod|
```

Will this work?